fast64_internal/sm64/sm64_collision.py

```diff
@@ -383,6 +383,8 @@ def exportCollisionCommon(obj, transformMatrix, includeSpecials, includeChildren
     tempObj, allObjs = duplicateHierarchy(obj, None, True, areaIndex)
     try:
         addCollisionTriangles(tempObj, collisionDict, includeChildren, transformMatrix, areaIndex)
+        if not collisionDict:
+            raise PluginError("No collision data to export, col export cancelled")
```

Comment:
If there is no collision data to export and it comes out as an error I don´t think you need to mention that the collision export was cancelled.